content/apps/introduction.md

```diff
@@ -0,0 +1,21 @@
+# What's an App ?
+
+::: warning
+The Apps are only available in the **6.0 version** and higher and for **SaaS** customers.
+:::
+
+Before you start developing an App, it's important to understand what Apps can do and how they interact with the PIM.
+
+An App is a service that can be easily connected to a PIM, directly from the PIM, 
+with a streamlined process of activation and configuration.
+
+An App can extend the PIM functionalities through the REST API, like a Connector, 
+by retrieving or updating data and integrate it with external services.
+
+The only constraint is that an App must have a single activation URL and should be hosted by its creator.
```

Comment:
```suggestion
Today, an App must have a single activation URL, be hosted by its creator, and use the OAuth2 to get API credentials. 
```

Reply:
> An App must have a single activation URL, be hosted by its creator and use OAuth2 to ask for API access.
?

Reply:
Yes
> An App must have a single activation URL, be hosted by its creator, and use OAuth2 to ask for API access.
They put a comma before "and" in English when there are more than 2 quoted things in a list